src/gettingstarted/own-code/create-project.md

```diff
@@ -0,0 +1,57 @@
+# Import your own code
+
+## Create a new project
+
+With the Platform.sh CLI now installed and configured to communicate with your projects, you can create a new project from the command line and connect it to your application.
+
+<asciinema-player src="/videos/asciinema/project-create.cast" preload=1 autoplay=1 loop=1></asciinema-player>
+
+1. **Create an empty project**
+
+    Type the command `platform create` in your terminal.
+
+    The CLI will then ask you to set up some initial project configurations:
+
+    * `Project title`: We need a unique name for each project, so title this one `My CLI Project`.
+
+    * `Region`: In general you will choose the region that is closest to where most of your site's traffic is coming from. Here, go ahead and begin typing `de-2.platform.sh` and the CLI will auto-complete the rest for you.
+
+    * `Plan`: The standard size will be fine for this guide, so type `1` and Enter.
+
+    * `Environments`: The `master` branch in the previous guide once built becomes the `Master` environment, the live production environment for your application. Additionally, you will receive other environments that will be useful for developing new features that you can view in your browser, but we will cover what those are and how to use them in the next guide.
```

Comment:
So far, this guide has been written on the assumption that the reader did not go through the template-based guide.  That's why the account creation and CLI installation process is repeated.  That means this paragraph should not refer back to a guide the user likely has not read.

Reply:
The wording here also feels clunky.  Suggestion: "Additionally, other branches may be activated as fully running environments for developing new features.  More on that later (link).  This value selects the maximum number of development environments the project will allow.  You can change this value later at any time."